File: Python/lab4/pc/GamingPc.py
from interfaces.PcProduct import PcProduct

class GamingPc(PcProduct):
    def show_pc_catalog(self):
        print("1. Збірка Nexus 22500грн\n"
              "2. Збірка Cyclope Limited 30000грн\n"
              "3. Збірка Optimum 22000грн\n"
              "4. Збірка Mega 42500грн\n"
              "5. Збірка Phoenix 17000грн\n")

File: Java/lab4/src/main/java/org/example/pc/GamingPc.java
package org.example.pc;

import org.example.interfaces.PcProduct;
public class GamingPc implements PcProduct {
    @Override
    public void showPcCatalog() {
        System.out.println("""
                1.Збірка Nexus 22500грн
                2.Збірка Cyclope Limited 30000грн
                3.Збірка Optimum 22000грн
                4.Збірка Mega 42500грн
                5.Збірка Phoenix 17000грн
                """);
    }
}
